src/main/java/mate/academy/internetshop/controller/ShoppingCartController.java

```diff
@@ -15,23 +15,22 @@
 
 @WebServlet("/shoppingCart")
```

Comment:
```suggestion
@WebServlet("/shoppingCart/products")
```
таким чином неймінг буде більш консистентний